### ISSUE TITLE
Cross gizmo

### ISSUE THREAD

**Issue by lubomirkurcak** [bevyengine/bevy#13868]
I often have a need for some kind of inoffensive gizmo to denote a point in space:

![cross](https://github.com/bevyengine/bevy/assets/29227697/f9c9f94a-823b-4b06-bcb1-0b0cb59c2a3e) ![cross_2d](https://github.com/bevyengine/bevy/assets/29227697/8c921878-226b-4f11-92f6-f32923b57643)

After copy-pasting code between projects I decided to publish a crate: https://crates.io/crates/bevy-cross-gizmo

Let me know if this is something people would use/find helpful. If so, I can create a pull request with code changes and changes to the bevy gizmo example to showcase it.

## Installation

```toml
[dependencies]
bevy-cross-gizmo = "0.13.2"
```

## Usage

```rust
use bevy::prelude::*;
use bevy_cross_gizmo::BevyCrossGizmo;

fn update(mut gizmos: Gizmos) {
    gizmos.cross(Vec3::ZERO, 0.3, Color::WHITE);
    gizmos.cross_2d(Vec3::ZERO, 0.3, Color::WHITE);
}
```

<details>
  <summary>Lib source code</summary>

```rust
use bevy::prelude::*;

pub trait BevyCrossGizmo {
    fn cross(&mut self, position: Vec3, size: f32, color: Color);
    fn cross_2d(&mut self, position: Vec2, size: f32, color: Color);
}

impl<'w, 's, T: GizmoConfigGroup> BevyCrossGizmo for Gizmos<'w, 's, T> {
    fn cross(&mut self, position: Vec3, size: f32, color: Color) {
        self.line(
            position + Vec3::new(size, 0.0, 0.0),
            position - Vec3::new(size, 0.0, 0.0),
            color,
        );
        self.line(
            position + Vec3::new(0.0, size, 0.0),
            position - Vec3::new(0.0, size, 0.0),
            color,
        );
        self.line(
            position + Vec3::new(0.0, 0.0, size),
            position - Vec3::new(0.0, 0.0, size),
            color,
        );
    }

    fn cross_2d(&mut self, position: Vec2, size: f32, color: Color) {
        self.line_2d(
            position + Vec2::new(size, 0.0),
            position - Vec2::new(size, 0.0),
            color,
        );
        self.line_2d(
            position + Vec2::new(0.0, size),
            position - Vec2::new(0.0, size),
            color,
        );
    }
}
```
</details>

**Comment by alice-i-cecile** [comment]
Yep that seems handy! Please PR this :)

**Comment by janhohenheim** [comment]
Implemented by #13883
